docs/adr/multi-protocol-xcall.md

```diff
@@ -0,0 +1,552 @@
+# XCall
+
+
+## Introduction
+xCall is a standard interface to make calls between different blockchain networks.
+
+## Protocol Overview
+![](https://hackmd.io/_uploads/SkBUxFhBh.png)
+
+
+## Prerequisites
+### Network Addresses
+XCall uses network address to refer to different addresses across many networks. A network consists of a network and an account section. A network address is represented as a string with network and account seperate by `/`. 
+A network is a unique id of a network and there can't be two networks with the same id connection to the same xCall network. 
+
```

Comment:
XCall uses network address to refer to different addresses across many networks. A "network address" consists of a network and an account section. A network address is represented as a string with "networkId" and account seperated by `/`. 
A networkId is a unique id of a network and there can't be two networks with the same id connection to the same xCall network.